docs/knowledge-base/monitoring/Grafana.mdx

```diff
@@ -0,0 +1,27 @@
+---
+id: Grafana
+title: Grafana
+sidebar_label: Grafana
+sidebar_position: 4
+sidebar_class_name: monitoring
+slug: /knowledge-base/monitoring/Grafana
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Grafana
+
+You can access Grafana at https://grafana.mapcolonies.net.
+
+Grafana is a visualization tool that allows you to query, visualize, and analyze metrics from multiple data sources, such as Prometheus, as well as trace data from Tempo.
+
+![grafana datasources](./img/grafana-datasources.png)
+
+Grafana enables you to create customizable and use powerful dashboards that give insights into system performance, resource usage, and alerting metrics.
+
+![grafana dashboard](./img/grafana-dashboard.png)
+
+In addition to visualizations, Grafana is also integrated with alerting capabilities, allowing you to trigger alerts when certain conditions are met in your metrics data.
```

Comment:
This is a Grafana feature that is not currently used. However, because we are explaining the whole monitoring and alerts stack, I think it should be mentioned here.

Reply:
OK